{"uid":"60fb77ff8aab0a89","name":"Successful login","fullName":"Tests.test_Login.TestLogin#test_2","historyId":"c2811b1721d8231bbb0fb40778ae099c","time":{"start":1728036812617,"stop":1728036813310,"duration":693},"status":"failed","statusMessage":"AssertionError: FAILURE: \ntest_Login.py:57 in test_2() -> check_with_screenshot(\nBaseClass.py:16 in check_with_screenshot() -> with check, allure.step(message):\nBaseClass.py:23 in check_with_screenshot -> assert cond\nAssertionError\n\n\n------------------------------------------------------------\nFailed Checks: 1","statusTrace":"FAILURE: \ntest_Login.py:57 in test_2() -> check_with_screenshot(\nBaseClass.py:16 in check_with_screenshot() -> with check, allure.step(message):\nBaseClass.py:23 in check_with_screenshot -> assert cond\nAssertionError\n\n\n------------------------------------------------------------\nFailed Checks: 1","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup_reporting_directories","time":{"start":1728036806640,"stop":1728036806641,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1728036811309,"stop":1728036811535,"duration":226},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: FAILURE: \ntest_Login.py:57 in test_2() -> check_with_screenshot(\nBaseClass.py:16 in check_with_screenshot() -> with check, allure.step(message):\nBaseClass.py:23 in check_with_screenshot -> assert cond\nAssertionError\n\n\n------------------------------------------------------------\nFailed Checks: 1","statusTrace":"FAILURE: \ntest_Login.py:57 in test_2() -> check_with_screenshot(\nBaseClass.py:16 in check_with_screenshot() -> with check, allure.step(message):\nBaseClass.py:23 in check_with_screenshot -> assert cond\nAssertionError\n\n\n------------------------------------------------------------\nFailed Checks: 1","steps":[{"name":"The app logo is visible after login","time":{"start":1728036812788,"stop":1728036812788,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"The app logo text is correct after login","time":{"start":1728036812808,"stop":1728036813174,"duration":366},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/home/runner/work/test-repo/test-repo/Common/BaseClass.py\", line 23, in check_with_screenshot\n    assert cond\n           ^^^^\n","steps":[],"attachments":[{"uid":"bf5c0adc87086f8b","name":"screenshot","source":"bf5c0adc87086f8b.png","type":"image/png","size":336923}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},{"name":"The items are visible after login","time":{"start":1728036813186,"stop":1728036813186,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1728036813311,"stop":1728036813378,"duration":67},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"critical"},{"name":"feature","value":"Login"},{"name":"parentSuite","value":"Tests"},{"name":"suite","value":"test_Login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1705-765"},{"name":"thread","value":"2458-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Tests.test_Login"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"driver","value":"''"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":0,"skipped":0,"passed":2,"unknown":0,"total":6},"items":[{"uid":"6a0ae715719f03a4","reportUrl":"https://Simeon-Zografov.github.io/test-repo/15//#testresult/6a0ae715719f03a4","status":"passed","time":{"start":1728036596824,"stop":1728036597160,"duration":336}},{"uid":"644f4c63f110baf8","reportUrl":"https://Simeon-Zografov.github.io/test-repo/14//#testresult/644f4c63f110baf8","status":"failed","statusDetails":"AssertionError: FAILURE: \ntest_Login.py:57 in test_2() -> check_with_screenshot(\nBaseClass.py:16 in check_with_screenshot() -> with check, allure.step(message):\nBaseClass.py:23 in check_with_screenshot -> assert cond\nAssertionError\n\n\n------------------------------------------------------------\nFailed Checks: 1","time":{"start":1728036064511,"stop":1728036065212,"duration":701}},{"uid":"e1ed69c495d0c5dd","reportUrl":"https://Simeon-Zografov.github.io/test-repo/13//#testresult/e1ed69c495d0c5dd","status":"failed","statusDetails":"AssertionError: FAILURE: \ntest_Login.py:57 in test_2() -> check_with_screenshot(\nBaseClass.py:16 in check_with_screenshot() -> with check, allure.step(message):\nBaseClass.py:23 in check_with_screenshot -> assert cond\nAssertionError\n\n\n------------------------------------------------------------\nFailed Checks: 1","time":{"start":1728035901906,"stop":1728035902617,"duration":711}},{"uid":"3865fb67ca403a0b","reportUrl":"https://Simeon-Zografov.github.io/test-repo/12//#testresult/3865fb67ca403a0b","status":"failed","statusDetails":"AssertionError: FAILURE: \ntest_Login.py:57 in test_2() -> check_with_screenshot(\nBaseClass.py:16 in check_with_screenshot() -> with check, allure.step(message):\nBaseClass.py:23 in check_with_screenshot -> assert cond\nAssertionError\n\n\n------------------------------------------------------------\nFailed Checks: 1","time":{"start":1728035651605,"stop":1728035652342,"duration":737}},{"uid":"861945b6be8eb47c","reportUrl":"https://Simeon-Zografov.github.io/test-repo/11//#testresult/861945b6be8eb47c","status":"passed","time":{"start":1728035272340,"stop":1728035272769,"duration":429}}]},"tags":[]},"source":"60fb77ff8aab0a89.json","parameterValues":["''"]}